NeuroID/src/main/java/com/neuroid/tracker/storage/NIDDataStoreManager.kt

```diff
@@ -140,24 +138,24 @@ internal object NIDDataStoreManagerImp : NIDDataStoreManager {
                     CONTEXT_MENU -> contextString = "meta=${event.metadata}"
                     ADVANCED_DEVICE_REQUEST -> contextString = "rid=${event.rid}, c=${event.c}"
                     LOG -> contextString = "m=${event.m}, ts=${event.ts}, level=${event.level}"
+                    CADENCE_READING_ACCEL -> contextString = "accel=${event.accel?.toString()}, gyro=${event.gyro?.toString()}"
                     else -> {}
                 }
 
                 NIDLog.d(
                     Constants.debugEventTag.displayName,
-                    "EVENT: ${event.type} - ${event.tgs} - ${contextString}"
+                    "EVENT: ${event.type} - ${event.tgs} - $contextString"
                 )
 
                 NeuroID.getInstance()?.captureIntegrationHealthEvent(event = event)
             }
 
             when (event.type) {
                 BLUR -> {
-                    NIDJobServiceManager.sendEventsNow(NIDLogWrapper())
+                    NeuroID.getInstance()?.nidJobServiceManager?.sendEventsNow()
```

Comment:
Does the behvaior of sending events in blur and close_session still need to be separated? Also to confirm, we are sending events even if the SDK is explicitly stopped,, but a close_session event happens?

Reply:
Yes we still want to send events immediately on blur and close session, the difference being close session should send events regardless of the SDK still running